.github/workflows/pin-requirements.yml

```
@@ -34,6 +30,20 @@ jobs:
         with:
           ref: main
 
+      - name: Get latest tag
+        id: latest_tag
+        run: echo "::set-output name=tag::$(git describe --tags --abbrev=0)"
+
+      - name: Set SIGSTORE_RELEASE_TAG and SIGSTORE_NEW_BRANCH
+        run: |
+          if [ -n "${{ inputs.tag }}" ]; then
```

Nitpick: forward these expansions through the env so that they don't unexpectedly expand into shell commands:

```suggestion
      - name: Set SIGSTORE_RELEASE_TAG and SIGSTORE_NEW_BRANCH
        env:
          INPUT_TAG: "${{ inputs.tag }}"
        run: |
          if [ -n "${INPUT_TAG}" ]; then
```

(and similarly for the others)